### PR TITLE
Update each access setting individually

### PR DESCRIPTION
**Pull request by masaball**

### CODE REVIEW

**Review by cjcolvar — APPROVED** [review]
Looks great!

**Comment by masaball** [comment]
I was doing another once over before merging and spotted an issue that would have kept the Special Access modal from working. I'd had it working since I demoed it, but I guess at some point those changes weren't committed or were accidentally overwritten by a later commit. I think things should be good now though.

File: app/jobs/bulk_action_jobs.rb [hunk 1]
```diff
@@ -197,24 +197,35 @@ def perform(target_id, subject_ids)
...
+        if save_field == "visibility"
+          media_object.visibility = collection.default_visibility
+        end
+        if collection.cdl_enabled? && save_field == "lending_period"
           media_object.lending_period = collection.default_lending_period
         end
 
         # Special access
-        if overwrite
+        if overwrite && save_field == "cdl"
```
Comment (cjcolvar) [inline app/jobs/bulk_action_jobs.rb:217]: Why is `save_field == "cdl"` important here?  I don't understand the connection between cdl and the special access fields.

**Review by cjcolvar — COMMENTED** [review]
I'm a bit confused by the CDL case.  Could we find a time to pair so you can walk me through it?

**Review by cjcolvar — APPROVED** [review]
Looks good!

**Pull request merged**